src/main/java/org/jitsi/jigasi/CallManager.java

```diff
@@ -623,4 +690,60 @@ public static synchronized void restartPool()
 
         threadPool = Executors.newFixedThreadPool(POOL_SIZE);
     }
+
+    /**
+     * A thread factory that creates daemon threads.
+     *
+     * @author Pawel Domas
+     */
+    private static class DaemonThreadFactory
```

Comment:
Is this a copy/paste from jicofo...?

Reply:
Please use `CustomizableThreadFactory` from jitsi-utils

Reply:
Yes.

Reply:
Nice, thanks.